### PR TITLE
Add docs for new ^ operator

### LINKED ISSUE #480: Document new power binop

### PR DESCRIPTION
**Pull request by brian-brazil**
Fixes #480

### CODE REVIEW

File: content/docs/querying/operators.md [hunk 1]
```diff
@@ -238,11 +239,13 @@ To get the 5 largest HTTP requests counts across all instances we could write:
...
+1. `^`
+2. `*`, `/`, `%`
+3. `+`, `-`
+4. `==`, `!=`, `<=`, `<`, `>=`, `>`
+5. `and`, `unless`
+6. `or`
 
 Operators on the same precedence level are left-associative. For example,
-`2 * 3 % 2` is equivalent to `(2 * 3) % 2`.
+`2 * 3 % 2` is equivalent to `(2 * 3) % 2`. However `^` is right associative,
```
Comment (juliusv) [inline content/docs/querying/operators.md]: "right associative" -> "right-associative"

File: content/docs/querying/operators.md [hunk 2]
```diff
@@ -238,11 +239,13 @@ To get the 5 largest HTTP requests counts across all instances we could write:
...
+so `2 * 3 ^ 2` is equivilent to `2 * (3 ^ 2)`.
```
Comment (juliusv) [inline content/docs/querying/operators.md]: equivilent -> equivalent

**Comment by juliusv** [comment]
:+1: otherwise

File: content/docs/querying/operators.md (hunk 2 above)
Comment (brian-brazil) [inline content/docs/querying/operators.md]: Done

File: content/docs/querying/operators.md (hunk 1 above)
Comment (brian-brazil) [inline content/docs/querying/operators.md]: Done

**Pull request merged**